rest-api-spec/src/main/resources/rest-api-spec/api/suggest.json

```diff
@@ -1,7 +1,7 @@
 {
   "suggest": {
     "documentation": "http://www.elastic.co/guide/en/elasticsearch/reference/master/search-suggesters.html",
-    "methods": ["POST", "GET"],
+    "methods": ["POST"],
```

Comment:
I removed the `GET` here to make sure we can test the deprecation warning [here](https://github.com/elastic/elasticsearch/pull/20305/files#diff-30d0b179e5896b66db3f509297d99cfdR33). Unfortunately, afaik the `warnings` message has to be an exact match.